lottie-compose/src/main/java/com/airbnb/lottie/compose/animateLottieCompositionAsState.kt

```diff
@@ -36,6 +36,7 @@ fun animateLottieCompositionAsState(
     clipSpec: LottieClipSpec? = null,
     speed: Float = 1f,
     iterations: Int = 1,
+    cancellationBehavior: LottieCancellationBehavior = LottieCancellationBehavior.Immediately,
```

Comment:
Can you add a param kdoc?

Reply:
ah right, I added the one from `animate`